Maple2.Server.Game/Manager/ItemBoxManager.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Database.Storage;
+using Maple2.Model;
+using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Metadata;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.Manager;
+
+public class ItemBoxManager {
+    #region Autofac Autowired
```

Comment:
don't need to pass these in, you can already access them via `session.ItemMetadata` and `session.TableMetadata`